<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="qtPasswordTextline">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qtAccountTextline">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入账号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
